debian/changelog

```diff
@@ -1,3 +1,30 @@
+raspberrypi-firmware (1:9.20230425+1-5.10.152-1+revpi10+1) buster; urgency=medium
```

Comment:
I would prefer an underscore for the counter? Just because of the plus signs in the debian revision. What do you think?

1:9.20230425_1-5.10.152-1+revpi10+1

Reply:
An underscore is not allowed in the upstream version:
> Upstream package name (-p): [-+.a-z0-9]{2,} 
https://www.debian.org/doc/manuals/debmake-doc/ch05.en.html#name-version